presto-hive/src/main/java/com/facebook/presto/hive/parquet/reader/ParquetMetadataReader.java

```diff
@@ -65,81 +65,88 @@ public static ParquetMetadata readFooter(FileSystem fileSystem, Path file, long
             throws IOException
```

Comment:
We might be able to get rid of this method if we can change the ParquetTester in presto-parquet module to use the new readFooter method.

Reply:
Thanks @nishantrayan. But I would prefer keeping this method just in case someone want to check the footer for once, so that they don't need to write code to handle the file open.